### PR TITLE
feat: expose consul limits to custom configuration

### PR DESCRIPTION
**Pull request by abarbare**
Expose the Consul `limits` with custom configuration https://www.consul.io/docs/agent/options.html#limits

### CODE REVIEW

**Comment by bbaassssiiee** [comment]
Thanks for the PR, due to another merge this now has conflicts. If you drop 3dc3f0f33a32fcc91cee7b32b9f96b43fb407b82 then I think we can rebase this in master.

File: .ansible-lint [hunk 1]
```diff
@@ -1,3 +1,2 @@
 skip_list:
   - '106'
-
```
Comment (bbaassssiiee) [inline .ansible-lint]: Please leave the newline at EOF

**Review by bbaassssiiee — CHANGES_REQUESTED** [review]
Thanks for the PR, due to another merge this now has conflicts. If you drop 3dc3f0f then I think we can rebase this in master.

File: .ansible-lint (hunk 1 above)
Comment (abarbare) [reply .ansible-lint]: Updated 👍

**Review by bbaassssiiee — APPROVED** [review]

**Pull request merged**